examples/meta/src/evaluation/multiclass_ovr_evaluation.sg.in

```diff
@@ -0,0 +1,23 @@
+File f_feats_train = csv_file("@SHOGUN_DATA@/classifier_4class_2d_linear_features_train.dat")
+File f_labels_train = csv_file("@SHOGUN_DATA@/classifier_4class_2d_linear_labels_train.dat")
+
+#![create_features]
+Features feats_train = features(f_feats_train)
+Labels labels_true = labels(f_labels_train)
+#![create_features]
+
+#![create_classifier]
+Machine svm= machine("MulticlassLibLinear", C=1.0, labels=labels_true)
+#![create_classifier]
+
+#![train_and_apply]
+svm.train(feats_train)
+Labels predicted_labels = svm.apply()
+RealVector labels = predicted_labels.get_real_vector("labels")
+#![train_and_apply]
+
+#![evaluate_accuracy]
+Evaluation binary_evaluator = evaluation("ContingencyTableEvaluation", type="ACCURACY")
+Evaluation ContingencyTableEvaluator = evaluation("MulticlassOVREvaluation", binary_evaluation=binary_evaluator)
```

Comment:
Could you stick to a naming convention (in this case snake_case) please?

Reply:
kk